indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
-<br>**Used in:** Indexing processing, page freshness scoring.<br>
-<br>
-<br>
+Details on what we index for PDFs can be found in our [PDF Metadata documentation here]({{site.baseurl}}/indexing/pdf-metadata.html).
+
+<div
+  class="usa-summary-box"
+  role="region"
+  aria-labelledby="summary-box-key-information"
+>
+  <div class="usa-summary-box__body">
+    <h3 class="usa-summary-box__heading" id="summary-box-key-information">
+      Supported Metadata
+    </h3>
+    <div class="usa-summary-box__text">
+      <ul class="usa-list">
+        <li>
+          <a class="usa-summary-box__link" href="#main-page-content">Main Content</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#title">Title</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#description">Description</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#tags--keywords">Tags</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#created-date">Created Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#changed-date">Changed Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#crawling-metadata">Crawling Metadata</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#thumbnail-image">Thumbnail Image</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#audience">Audience</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#content-type">Content Type</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#searchgov-custom-fields">Search.gov Custom Fields</a> 
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+### Fields by Type
+
+|[Standard Fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)|[OpenGraph Fields](https://ogp.me/)|[DublinCore Fields](https://www.dublincore.org/)|
+|:----|:----|:----|:----|
+|[`<main>`](#main-page-content)|[`og:title`](#title)|[`dcterms.keywords`](#tags--keywords)|
+|[`<title>`](#title)|[`og:description`](#description)|[`dc.subject`](#tags--keywords)|
+|[`description`](#description)|[`article:section`](#tags--keywords)|[`dcterms.subject`](#tags--keywords)|
+|[`keywords`](#tags--keywords)|[`article:tag`](#tags--keywords)|[`dcterms.audience`](#audience)|
+|[`robots`](#crawling-metadata)|[`article:published_time`](#created-date)|[`dc.type`](#content-type)|
+||[`article:modified_time`](#changed-date)|[`dcterms.type`](#content-type)|
+||[`og:image`](#thumbnail-image)|[`dc.date`](#created-date)|
+|||[`dc.date.created`](#created-date)|
+|||[`dcterms.created`](#created-date)|
+
+
+### Main Page Content
+We look for the following field:
```

Comment:
Looking at the code, there are also three additional fall-back options that are consulted (https://github.com/GSA/search-gov/blob/main/app/models/html_document.rb#L125).  This method grabs the first of these options that has text content:

1. The `<main>`, as you've described here;
2. Content inside the _first_ tag that has a `role = 'main'`;
3. All content in `<body>`; and, finally,
4. All content in `<html>`.

Not sure how often any of these fallbacks are consulted, but that is what will happen if main is absent.

Reply:
I agree we should mention the fallbacks if there isn't a `<main>`

Reply:
added